index.html

```diff
@@ -85,6 +86,176 @@ <h1>Hey There. I’m Akhtar</h1>
             </li>
         </ul>
     </main>
+    <section class="works">

```

Comment:
- [ ] **Please make sure the section uses the colors (background and font) defined for the template**

The the colors for project section on your website do not match with the original design. Please fix this. 

Cloned Page|Original Design
:----------------:|:------------------:
![image](https://user-images.githubusercontent.com/59168430/120896004-25896c80-c63d-11eb-902b-f9496b3ddbf6.png)|![image](https://user-images.githubusercontent.com/59168430/120896013-32a65b80-c63d-11eb-8185-084fab8809a0.png)

Reply:
Thank you for the review. The colors have been amended.

Reply:
This section looks similar to the design template now :heavy_check_mark: